src/main/java/org/openbase/bco/bcozy/controller/powerterminal/chartcontroller/TilesFxChartController.java

```diff
@@ -34,8 +36,10 @@
     public ScheduledFuture enableDataRefresh(long interval, ChartStateModel chartStateModel) {
         ScheduledFuture refreshSchedule = null;
         try {
-            refreshSchedule = GlobalScheduledExecutorService.scheduleAtFixedRate(() -> Platform.runLater(() -> updateChart(chartStateModel)),
-                    10, interval, TimeUnit.MILLISECONDS);
+            refreshSchedule = GlobalScheduledExecutorService.scheduleAtFixedRate(() -> {
+                List<ChartData> data = PowerTerminalDBService.getAverageConsumption(chartStateModel.getDateRange(), chartStateModel.getSelectedConsumer());
```

Comment:
To improve the performance, check if the chart pane is currently displayed and only then perform the update task. Otherwise, even when only the location map is displayed, the db is continuously queried.

Reply:
Fixed in 31576ec